stock_picking_package_info/models/product_packaging_template.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from openerp.addons import decimal_precision as dp
+
+
+class ProductPackagingTemplate(models.Model):
+    _name = 'product.packaging.template'
+    _description = 'Product Packaging Template'
+    _rec_name = 'packaging_template_name'
+
+    packaging_template_name = fields.Char(required=True)
+    type = fields.Selection([
```

Comment:
`tmpl` is a common abbreviation you find across Odoo, but OK then to put `template_type`. The word is not a keyword, but a function name that is in the base: http://stackoverflow.com/questions/10568087/is-it-safe-to-use-the-python-word-type-in-my-code